File: Assignment_5/program1.c
// Write a program which accept number from user and print that number of $ & * on screen

/*
    INPUT : 5
    OUTPUT : $  *  $  *  $  *  $  *  $  *
    
    INPUT : 3
    OUTPUT : $  *  $  *  $  * 

    INPUT : -3
    OUTPUT : $  *  $  *  $ * 

*/
//Time Complexity = O(N)

#include<stdio.h>

void Pattern (int iNo)
{

    if (iNo < 0)
    {
        iNo = -iNo;
    }
    int iCnt = 0;

    for(iCnt = 1; iCnt <= iNo; iCnt++)
    {
        printf(" $  * ");
    }

}

int main()
{
    int iValue = 0;

    printf("Enter number :");
    scanf("%d",&iValue);

    Pattern(iValue);

    return 0;
}

File: Assignment_48/program1.java
// Write Java program which accept String from user and display below pattern. 
/*
    Input : Hello
    Output : H e l l o
             H e l l o
             H e l l o
             H e l l o
             H e l l o
  
 */

import java.util.Scanner;

class Pattern 
{ 
    public void Pattern(String str) 
    { 
        char[] strnew = str.toCharArray();

        int i = 0, j = 0;
        for(i = 0; i < strnew.length; i++)
        {
            for(j = 0; j < strnew.length; j++)
            {
                System.out.print(strnew[j] +"\t");
            }

            System.out.println();
        }

    } 
}

class program1 
{
    public static void main(String[] A) 
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter string");
        String str = sobj.nextLine();

        Pattern pobj = new Pattern();
        pobj.Pattern(str);
        
    }
    
}
